lib/Twig/Template.php

```diff
@@ -321,33 +321,6 @@ public function getBlocks()
     }
 
     /**
-     * Returns the template source code.
-     *
-     * @return string|null The template source code or null if it is not available
-     */
-    public function getSource()
```

Comment:
I mean, I understand the BC concern, but I don't think we need to care about it in this case. Nobody is expected to replace `Twig_Node_Module` in such a drastic way. Am I missing something?